samples/bot-adaptive-card-actions/nodejs/Readme.md

```diff
@@ -0,0 +1,133 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send Adaptive Card actions using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "12/27/2022 12:30:17 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-adaptivecard-actions-nodejs
+---
+# Get Adaptive Card and Suggested Actions
+
+This sample shows the feature where user can send suggested actions using bot.
```

Comment:
This sample shows the feature where user can send adaptive card with different actions using bot.

Reply:
fixed